data/json/effects_on_condition/scenario_specific_eocs.json

```diff
@@ -308,5 +308,11 @@
     "update_mapgen_id": "SCENARIO_SMOKE",
     "method": "json",
     "object": { "place_fields": [ { "field": "fd_smoke", "x": [ 3, 20 ], "y": [ 3, 20 ], "intensity": [ 1, 3 ], "repeat": 60 } ] }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_scenario_clear_weather",
+    "eoc_type": "SCENARIO_SPECIFIC",
+    "effect": [ { "math": [ "weather('humidity')", "=", "0" ] }, { "math": [ "weather('pressure')", "=", "1050" ] }, "next_weather" ]
```

Comment:
try to delay next weather by 1 second?

Reply:
Not sure I'm doing this correctly, but I'm only getting the first u_message "EOC_scenario_clear_weather". Also I spawned a hygrometer and checked humidity and it doesn't read 0%.

```json
  {
    "type": "effect_on_condition",
    "id": "EOC_next_weather",
    "effect": [ { "u_message": "EOC_next_weather" }, "next_weather" ]
  },  
  {
    "type": "effect_on_condition",
    "id": "EOC_scenario_clear_weather",
    "eoc_type": "SCENARIO_SPECIFIC",
    "effect": [
      { "math": [ "weather('humidity')", "=", "0" ] },
      { "math": [ "weather('pressure')", "=", "1050" ] },
      { "u_message": "EOC_scenario_clear_weather" },
      { "run_eocs": "EOC_next_weather", "time_in_future": "1 seconds" }
    ]
  }
```

Reply:
I fixed the issue in another way and I'm resolving this so PR can be merged, but setting things with `{ "math": [ "weather('humidity')", "=", "0" ] },` is still an issue.